NanoAnalysis/python/nanoZZ4lAnalysis.py

```diff
@@ -193,14 +196,19 @@
     insertBefore(reco_sequence, 'lepFiller', getEleScaleRes(LEPTON_SETUP, DATA_TAG, IsMC, overwritePt=True))
 
 # Update of JetId from manual recipe for NanoAOD v12
-if NANOVERSION == 12:
-    from ZZAnalysis.NanoAnalysis.jetIdUpdate import *
-    insertBefore(reco_sequence, 'jetFiller', jetIdUpdate())
+#if NANOVERSION == 12:
 
+from ZZAnalysis.NanoAnalysis.jetIdUpdate import *
+insertBefore(reco_sequence, 'jetFiller', jetIdUpdate())
+#if NANOVERSION == 15:
+#    from PhysicsTools.NATModules.modules.jetidJSON import *
+#    insertBefore(reco_sequence, 'jetFiller', )
 # Add jet corrections for Run 3
 if APPLYJETCORR and LEPTON_SETUP >=2022 :
+    print("YOU ARE NOT RUNNING ON NANOAODV12")
```

Comment:
What does this mean? APPLYETCORR is True in [2023 samples](https://github.com/CJLST/ZZAnalysis/blob/Run3/NanoAnalysis/test/prod/samplesNano_2023postBPix_MC.csv) for example, which are v12.